data/json/professions.json

```diff
@@ -2704,7 +2711,10 @@
           "mbag",
           "nomex_socks",
           "water_clean",
-          "pocketwatch"
+          "pocketwatch",
```

Comment:
Why would I firefighter carry `"id": "pocketwatch",
    "type": "TOOL",
    "name": { "str": "pocket watch", "str_pl": "pocket watches" },
    "description": "An old-fashioned pocket watch.  Tells the time and looks good doing it.  Can also be disassembled into some useful parts.",`?